README.md

```diff
@@ -1,6 +1,16 @@
 # Mbed TLS Examples on Mbed OS
 
-This repository contains a collection of Mbed TLS example applications based on Mbed OS. Each subdirectory contains a separate example meant for building as an executable.
+This repository contains following Mbed TLS example applications based on Mbed OS. Each subdirectory contains a separate example meant for building as an executable. Please browse to the subdirectories for specific documentation.
```

Comment:
'This repository contains following Mbed TLS example applications' doesn't make sense.
 
I think you need to revert to the original line:
> This repository contains a collection of Mbed TLS example applications based on Mbed OS.